pkg/syncthing/process.go

```diff
@@ -14,12 +14,37 @@
 package syncthing
 
 import (
+	"runtime"
 	"time"
 
 	"github.com/shirou/gopsutil/process"
 )
 
 func terminate(p *process.Process, wait bool) error {
+	if runtime.GOOS == "windows" {
```

Comment:
I think the logic is cleaner if there is a function to get the children, then kill the parent, then kill the children.
If `runtime.GOOS != "windows"`, children is `[]`

Reply:
If we call it on linux children won't be `[]`, it will be the same as it's on windows and you will have one more call to make (the call to get children). I also thought to be OS agnostic but having one more process call is not very suitable

Reply:
What I mean is to artificially return an empty list if it is not windows. We can call it `getZombieChildren` to make it clearer